app/views/users/show.html.erb

```diff
@@ -1,12 +1,32 @@
-<div class="container">
-  <h2><%= @user.email %></h2>
-    <% @wikis.each do |wiki| %>
-    <ul>
-      <li><%= link_to wiki.title, wiki %></li>
-    </ul>
-    <% end %>
-</div>
+<h3><strong>User: </strong><%= @user.email %></h3>
+<br />
 <br />
 <div class="container">
-  <%= render partial: "charges/form", locals: {stripe_btn_data: @stripe_btn_data} %>
+  <div class="row">
+    <div class="col-sm-4">
+      <% @wikis.each do |wiki| %>
+        <div class="row">
+          <ul>
+            <li><%= link_to wiki.title, wiki %></li>
+          </ul>
+        </div>
+      <% end %>
+    </div>
+    <div class="col-sm-4">
+      <div class="col-md-4">
+        <% @wiki.collaborators.each do |wiki| %>
```

Comment:
@bjblock - I know I'm close here but for some reason, I can't get the actual Wiki's that I'm a collaborator on to show. The page loads fine, I double checked to make sure I was actually a collaborator on a few wikis and it still loads the page but without showing the wikis. Any hints?